app/src/main/java/com/github/polypoly/app/game/Trophy.kt

```diff
@@ -0,0 +1,39 @@
+package com.github.polypoly.app.game
+
+/**
+ * A trophy that can be win during a game or at the end of a game to reward a particular
+ * success of the player.
+ */
+class Trophy(
```

Comment:
FYI this can be a
`data class` in Kotlin. It's the equivalent of a `record` in Java.